lib/rubocop/cop/style/redundant_conditional.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Style
+      # This cop checks for redundant returning of true/false in conditionals
+      #
+      # @example
+      #   # bad
+      #   x == y ? true : false
+      #
+      #   # bad
+      #   if x == y
+      #     true
+      #   else
+      #     false
+      #   end
+      #
+      #   # good
+      #   x == y
+      #
+      #   # bad
+      #   x == y ? false : true
+      #
+      #   # good
+      #   x != y
+      class RedundantConditional < Cop
+        COMPARISON_OPERATORS = RuboCop::AST::Node::COMPARISON_OPERATORS
+
+        MSG = 'Returning true/false from a conditional is unnecessary.'.freeze
```

Comment:
Done. I haven't implemented interpolation as I figure we can't guarantee that the conditional will be short and a single-line string, which would presumably result in messy output?